src/frontend/dash/circlePacker.ts

```diff
@@ -143,9 +143,19 @@ export class ClockCircle extends Circle{
                 document.querySelectorAll('.bustime.west').forEach(timeElems.push);
 
                 for(let busSignIndex = 0; busSignIndex < timeElems.length; busSignIndex++) {
-                    let minutesTill = Math.max(0,Math.round((arrivals[busSigns[busSignIndex >> 1]][busSignIndex & 1] - Date.now())/1000/60));
+                    let arrivalsCardinal;
+                    if(busSignIndex < 2)
+                        arrivalsCardinal = arrivals[busSigns[0]];
+                    else
+                        arrivalsCardinal = arrivals[busSigns[1]];
```

Comment:
I would support a ternary operator here

Reply:
Zach told me something, and I took it very literally

Reply:
I suggested that you should avoid using ternaries in the way that they are used below. Ternaries are not inherently problematic, the point is to have code that is straightforward to read
```ts
timeElems[0].classList[minutesTill <= 5 ? "add" : "remove"]("soonish")
```

For this line that Azalea has commented on, I really don't think it matters if it's a ternary or not